packaging/el8/cloud.spec

```diff
@@ -17,6 +17,8 @@
 
 %define __os_install_post %{nil}
 %global debug_package %{nil}
+%global __requires_exclude *libc.so.*
```

Comment:
JFYI - I couldn't figure out how to exclude rpm to indirectly depend on libc, any feedback/ideas are welcome. cc @NuxRo @weizhouapache @DaanHoogland @JoaoJandre and others.

The workaround is currently that you force install cloudstack-common (where libc is a dependency) using `rpm -ivh --no-deps` and then rest pkgs like cloudstack-agent etc install okay.

Reply:
from https://docs.fedoraproject.org/en-US/packaging-guidelines/AutoProvidesAndRequiresFiltering/
it looks double backslash is required for `.`

maybe like this (to exclude libc.so.X) ?
```
%global __requires_exclude libc\\.so\\..*
```

Reply:
Thanks @weizhouapache I've applied your suggestion, will test.